exercises/kindergarten-garden/kindergarten_garden_test.exs

```diff
@@ -0,0 +1,77 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("kindergarten_garden.exs")
```

Comment:
Usually the file name matches the module contained inside. This should be `garden.exs` or the module should be renamed.